Aspire Budgeting/Views/Dashboard/DashboardCardsListView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  DashboardCardsListView.swift
+//  Aspire Budgeting
+//
+
+import SwiftUI
+
+struct DashboardCardsListView: View {
+
+  let cardViewItems: [DashboardCardView.DashboardCardItem]
+
+  var body: some View {
+    GeometryReader {g in
+      ScrollView {
+        VStack {
+          ForEach(0..<self.cardViewItems.count) { idx in
+            GeometryReader { geo in
+              BaseCardView<DashboardCardView>(
+                       minY: g.frame(in: .global).minY,
+                       curY: geo.frame(in: .global).minY,
+                baseColor: colorFor(idx: idx)) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/1531d43d43f9ff835e554eb2943298d8b02e0dbb/Aspire Budgeting/Views/Dashboard/DashboardCardsListView.swift#L21"></span>Function parameters should be aligned vertically if they’re in multiple lines in a method call.<br />
<code>vertical_parameter_alignment_on_call</code> <code>DashboardCardsListView.swift:21</code></td>
    </tr>
  </tbody>
</table>